go/oasis-test-runner/oasis/runtime.go

```diff
@@ -36,6 +37,7 @@ type Runtime struct { // nolint: maligned
 	teeHardware node.TEEHardware
 	mrEnclaves  []*sgx.MrEnclave
 	mrSigner    *sgx.MrSigner
+	version     uint64
```

Comment:
Where is this used?

Reply:
Oops, this should go to `RuntimeCfg` struct. Removed.